### ISSUE TITLE
[anima fast]显存占用问题

### ISSUE THREAD

**Issue by qweq2004** [wochenlong/lora-scripts-next#214]
<img width="2072" height="1372" alt="Image" src="https://github.com/user-attachments/assets/65404c58-b06c-4906-818e-6fb47bc0494f" />
如图所示，我的显存明明还有剩余，但是还是会占用内存，而普通的lora训练就能吃满，下面是参数
model_train_type = "anima-lora-fast"
lora_type = "lora"
pretrained_model_name_or_path = "H:/AI/SD-Trainer-v2.9.1/SD-Trainer/sd-models/anima/anima-base-v1.0.safetensors"
vae = "H:/AI/SD-Trainer-v2.9.1/SD-Trainer/sd-models/anima/qwen_image_vae.safetensors"
qwen3 = "H:/AI/SD-Trainer-v2.9.1/SD-Trainer/sd-models/anima/qwen_3_0.6b_base.safetensors"
method = "lora"
methods_subdir = "gui-methods"
qwen3_max_token_length = 512
timestep_sampling = "shift"
discrete_flow_shift = 3
attn_mode = ""
torch_compile = true
static_token_count = 9216
compile_mode = "blocks"
train_data_dir = "H:/AI/SD-Trainer-v2.9.1/train/test"
cache_latents = false
cache_latents_to_disk = false
cache_text_encoder_outputs = false
cache_text_encoder_outputs_to_disk = false
skip_cache_check = false
caption_extension = ".txt"
resolution = "1536,1536"
enable_bucket = true
min_bucket_reso = 256
max_bucket_reso = 8192
bucket_reso_steps = 64
bucket_no_upscale = false
output_name = "Umineko"
output_dir = "./output"
save_model_as = "safetensors"
save_precision = "fp16"
save_every_n_epochs = 1
save_state = true
logging_dir = "./logs/anima_fast"
max_train_epochs = 8
train_batch_size = 8
dataset_repeats = 1
gradient_checkpointing = true
gradient_accumulation_steps = 1
seed = 42
learning_rate = 0.0005
unet_lr = 0.0005
text_encoder_lr = 0.00001
lr_scheduler = "cosine_with_restarts"
lr_warmup_steps = 0
lr_scheduler_num_cycles = 1
optimizer_type = "AdamW8bit"
enable_preview = false
network_module = "networks.lora_anima"
network_dim = 32
network_alpha = 16
gpu_ids = [ "0" ]

**Comment by wochenlong** [comment]
这是旧版本 Anima Fast 的显存占用反馈。当前先关闭，不回溯旧版本专门修复；如果在当前 v3.1.0 仍能稳定复现，请附显卡型号、当前配置和日志重新打开或提交新 Issue。
